src/components/DarkModeSwitch.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from "react";
+import { TbSun, TbMoon } from "react-icons/tb";
+
+const iconDiv =
+  "absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 transform cursor-pointer transition-opacity duration-200";
+
+function DarkModeSwitch() {
+  //Set light theme by default
+  const [theme, setTheme] = useState("dark");
+
+  useEffect(() => {
+    //Set theme on device
+    if (typeof window !== "undefined") {
+      if (localStorage.theme === "dark" || !("theme" in localStorage)) {
```

Comment:
setting and saving the selected theme in local storage is ok, but we should be able to detect the current prefer of the user via matchmedia. You can have a look at this [article](https://blog.logrocket.com/dark-mode-in-react-an-in-depth-guide/) for some idea if not familiar with match media yet.

Reply:
I used matchmedia at first but Sir Warren asked to make dark mode as default [here].(https://github.com/web3phl/directory/pull/62#pullrequestreview-1096679545)
I will try to apply the [useLocalStorage](https://usehooks-ts.com/react-hook/use-local-storage) hook.
Thank you for the recommendations and inputs.

Reply:
Good catch @jnebab for this one!

Reply:
> I used matchmedia at first but Sir Warren asked to make dark mode as default [here].([#62 (review)](https://github.com/web3phl/directory/pull/62#pullrequestreview-1096679545)) I will try to apply the [useLocalStorage](https://usehooks-ts.com/react-hook/use-local-storage) hook. Thank you for the recommendations and inputs.

My bad, @jnebab has point, if the user's system prefers dark mode we should follow that instead. If not we should follow too.

Reply:
> I used matchmedia at first but Sir Warren asked to make dark mode as default [here].([#62 (review)](https://github.com/web3phl/directory/pull/62#pullrequestreview-1096679545)) I will try to apply the [useLocalStorage](https://usehooks-ts.com/react-hook/use-local-storage) hook. Thank you for the recommendations and inputs.

my apologies too, didn't read the previous comments before I made my review. if you can do the change, we will appreciate it very much. Thank you @rgvillanueva28